test_settings.py

```diff
@@ -68,6 +68,7 @@ def root(*args):
     'waffle',
     'test_utils.test_app',
     'eventtracking.django.apps.EventTrackingConfig',
+    'event_routing_backends',
```

Comment:
I was able to get tests working by moving completion_aggregator to the bottom of this list, though you could probably just move ERB above it here.